<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="windowsMenu">
    <property name="title">
     <string>Відкрити</string>
    </property>
    <addaction name="settingsAction"/>
    <addaction name="plotAction"/>
    <addaction name="tableAction"/>
   </widget>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="openAction"/>
    <addaction name="resetAction"/>
    <addaction name="printAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="windowsMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="settingsAction">
   <property name="text">
    <string>Налаштування</string>
   </property>
  </action>
  <action name="plotAction">
   <property name="text">
    <string>Графік</string>
   </property>
  </action>
  <action name="tableAction">
   <property name="text">
    <string>Таблиця</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>Зберегти</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>Зберегти як</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>Відкрити</string>
   </property>
  </action>
  <action name="resetAction">
   <property name="text">
    <string>Скинути значення</string>
   </property>
  </action>
  <action name="printAction">
   <property name="text">
    <string>Друк</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>settingsAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>triggerSettingsAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plotAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>triggerPlotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>triggerTableAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>triggerSaveAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAsAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>triggerSaveAsAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>triggerOpenAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>triggerResetAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>triggerPrintAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>releaseCalculationButton()</slot>
  <slot>triggerSettingsAction()</slot>
  <slot>triggerPlotAction()</slot>
  <slot>triggerTableAction()</slot>
  <slot>triggerSaveAction()</slot>
  <slot>triggerSaveAsAction()</slot>
  <slot>triggerOpenAction()</slot>
  <slot>triggerResetAction()</slot>
  <slot>triggerPrintAction()</slot>
 </slots>
</ui>
